### PR TITLE
Allow user to choose menu when creating session

### PR DESCRIPTION
**Pull request by acchiang**
image

### CODE REVIEW

File: client/src/pages/OrderScreen.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import TextIcon from "components/TextIcon";
 import TopTitleBar from "components/TopTitleBar";
 
 const serverURL = "http://localhost:9000";
-const menuId = "6103677a11c316178047f1f1";
+const DEFAULT_MENU_ID = "6103677a11c316178047f1f1";
```
Comment (christylo) [inline client/src/pages/OrderScreen.js:16]: will we be keeping this hard coded or changing it in the future

Comment (acchiang) [reply client/src/pages/OrderScreen.js:16]: hard coded unless we change the default menu

Comment (kozr) [reply client/src/pages/OrderScreen.js:16]: why is there a hard coded default menu? shouldn't we pull the list of menus from the db  by Thursday?

Comment (kozr) [reply client/src/pages/OrderScreen.js:16]: cause technically it's supposed to be production ready

**Pull request merged**